verde/utils.py

```diff
@@ -261,6 +261,137 @@ def maxabs(*args, nan=True, percentile=100):
         return nppercentile(combined_array, percentile)
 
 
+def minmax(*args, nan=True, min_percentile=0, max_percentile=100):
+    """
+    Calculate the minimum and maximum values (or percentiles) of the given
+    array(s).
+
+    Use this to set the limits of your colorbars for non-diverging data.
+
+    Parameters
+    ----------
+    args
+        One or more arrays. If more than one are given, a minimum and maximum
+        will be calculated across all arrays.
+    nan : bool, optional
+        If True, will use the ``nan`` version of numpy functions to ignore
+        NaNs.
+    min_percentile: float
+        Return the supplied percentile (0 to 100) instead of the minimum value
+        of the arrays. Defaults to 0, giving the minimum value.
+    max_percentile : float
+        Return the supplied percentile (0 to 100) instead of the maximum value
+        of the arrays. Defaults to 100, giving the maximum value.
+
+    Returns
+    -------
+    min, max : float
+        The minimum and maximum (or percentile) values across all arrays.
+
+    Examples
+    --------
+
+    >>> result = minmax((1, -10, 25, 2, 3))
+    >>> tuple(map(float, result))
```

Comment:
We can replace these with a `print` statement that is less cryptic for users than using `map`:
```suggestion
    >>> print(result)
```